events/apigw.go

```diff
@@ -285,6 +285,12 @@ type APIGatewayCustomAuthorizerResponse struct {
 	UsageIdentifierKey string                           `json:"usageIdentifierKey,omitempty"`
 }
 
+// APIGatewayV2CustomAuthorizerSimpleResponse represents the simple format of an API Gateway V2 authorization response.
+type APIGatewayV2CustomAuthorizerSimpleResponse struct {
+	IsAuthorized bool                   `json:"isAuthorized"`
+	Context      map[string]interface{} `json:"context"`
```

Comment:
Did you get a chance to verify this works as expected with a context set and without it set (using the default value). I just wonder what happens if `Context` is nil (`null` in json) and/or an empty map (`{}` in json)

Reply:
Yes, already using it in production for a while. Works with or without context. So the API accepts the input. But I could check what’s exactly generated and if it makes sense to add `omitempty`.

Reply:
It might make sense and save just a few bytes over the long run so if you could check that would be great

Reply:
Hey @harrisonhjones, I just added the tests. Pls take a look if I covered everything.

Reply:
BTW, I also added the `omitempty` flag, as the results are more reasonable.